examples/reference/panes/HTML.ipynb

```diff
@@ -97,6 +97,65 @@
    "source": [
     "html_pane.styles = dict(html_pane.styles, border='2px solid red')"
    ]
+  },
```

Comment:
An alternative to documenting this would be to add a `Iframe` pane. Believe I asked for this many years ago but was rejected. I see it needed from time to time and it takes time to construct the code. Its friction.

Reply:
I don't remember rejecting the idea.